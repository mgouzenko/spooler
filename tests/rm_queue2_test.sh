addqueue tests/test_dir/cat
rmqueue 1
