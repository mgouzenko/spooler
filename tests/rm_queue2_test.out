tests/test_dir/cat: Y 1
1: Y
